### ISSUE TITLE
Support for Audio and Video Bags/Work Types

### ISSUE THREAD

**Issue by ntallman** [psu-libraries/cho#892]
* Digitized time-based media may or may not have its components (cassettes, reels, etc.) joined together in a single service or preservation file.
* Digitized time-based media may or may not include a XML file that includes administrative/technical information about the digitization process, particularly if vended.
* Digitized time-based media should include image(s) of the original analog media. A front or front and back image may be included. These images are typically JPGs not TIFs.

Questions:
* What file set does the metadata belong to? Are they preservation files? Are identifiers assigned?
* What file set do the front/back images belong to? Are they preservation files? Are identifiers assigned?
* Do either the metadata files or carrier images get represented in the CSV metadata? If not part of the representative file set, yes?

Blocked by #649, which may allow files with an `_front` or `_back` suffix/use type.
